widgetry/src/widgets/spinner.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
 };
 
 // Manually tuned
-const TEXT_WIDTH: f64 = 80.0;
+const TEXT_WIDTH: f64 = 100.0;
```

Comment:
![Screenshot from 2021-04-29 13-26-26](https://user-images.githubusercontent.com/1664407/116614425-c4a89f00-a8ee-11eb-9034-a5ac9d8e7be0.png)
Needed to fit this better